File: C++/FizzBuzz.cpp
#include <iostream>
using namespace std;

int main()
{
    for (int x=1; x <= 30; x++){
        if (x % 15 == 0) {
                cout << "FizzBuzz" << endl;
                continue;
        };
        if (x % 3 == 0) {
            cout << "Fizz" << endl;
            continue;
        };
        if (x % 5 == 0) {
            cout << "Buzz" << endl;
            continue;
        };
        cout << x << endl;
    };
    return 0;
}

File: Python/fizzbuzz.py
'''
     Coding Club Quick Test FizzBuzz - Zac Reid

 - How it works?

 - If it is divisable by 3 say Fizz
 - If it is divisable by 5 say Buzz
 - If both of the above statements are true say FizzBuzz
 
 - Example output 1, 2, Fizz, 4, Buzz, Fizz, 7, 8, Fizz, Buzz, 11, Fizz, 13, 14, FizzBuzz
 (1 <= input <= 100)
 '''
def fizzBuzz(num):
    if num % 15 == 0:
        return 'FizzBuzz'

    if num % 3 == 0:
        return 'Fizz'

    if num % 5 == 0:
        return 'Buzz'

    return num
    
for x in range(1000):
    print(fizzBuzz(x))